Emby.Server.Implementations/LiveTv/Listings/XmlTvListingsProvider.cs

```diff
@@ -66,24 +68,38 @@ private async Task<string> GetXml(string path, CancellationToken cancellationTok
                 return UnzipIfNeeded(path, cacheFile);
             }
 
-            _logger.LogInformation("Downloading xmltv listings from {path}", path);
-
-            string tempFile = await _httpClient.GetTempFile(new HttpRequestOptions
-            {
-                CancellationToken = cancellationToken,
-                Url = path,
-                Progress = new SimpleProgress<double>(),
-                // It's going to come back gzipped regardless of this value
-                // So we need to make sure the decompression method is set to gzip
-                DecompressionMethod = CompressionMethod.Gzip,
-
-                UserAgent = "Emby/3.0"
-
-            }).ConfigureAwait(false);
+            _logger.LogInformation("Downloading xmltv listings from {Path}", path);
 
             Directory.CreateDirectory(Path.GetDirectoryName(cacheFile));
 
-            File.Copy(tempFile, cacheFile, true);
+            using (var res = await _httpClient.SendAsync(
+                new HttpRequestOptions
+                {
+                    CancellationToken = cancellationToken,
+                    Url = path,
+                    Progress = new SimpleProgress<double>(),
+                    // It's going to come back gzipped regardless of this value
+                    // So we need to make sure the decompression method is set to gzip
+                    DecompressionMethod = CompressionMethod.Gzip,
+
+                    UserAgent = "Emby/3.0"
```

Comment:
Is there a reason to stick to emby here?

Reply:
Removed it